napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -54,22 +39,53 @@ class QtImageControls(QtBaseImageControls):
+
+    Controls attributes
+    -------------------
+    blendComboBox : qtpy.QtWidgets.QComboBox
+        Dropdown widget to select blending mode of layer.
+    blendLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the blending combobox widget.
+    opacitySlider : qtpy.QtWidgets.QSlider
+        Slider controlling opacity of the layer.
+    opacityLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the opacity slider widget.
+    contrastLimitsSlider : superqt.QRangeSlider
+        Contrast range slider widget.
+    autoScaleBar : qtpy.QtWidgets.QWidget
+        Widget to wrap push buttons related with the layer auto-contrast funtionality.
+    gammaSlider : qtpy.QtWidgets.QSlider
+        Gamma adjustment slider widget.
+    colormapWidget : qtpy.QtWidgets.QWidget
+        Widget to wrap combobox and label widgets related with the layer colormap attribute.
     interpComboBox : qtpy.QtWidgets.QComboBox
         Dropdown menu to select the interpolation mode for image display.
     interpLabel : qtpy.QtWidgets.QLabel
         Label for the interpolation dropdown menu.
-    isoThresholdSlider : qtpy.QtWidgets.QSlider
-        Slider controlling the isosurface threshold value for rendering.
-    isoThresholdLabel : qtpy.QtWidgets.QLabel
-        Label for the isosurface threshold slider widget.
+    depictionComboBox : qtpy.QtWidgets.QComboBox
+        ComboBox controlling current depiction value of the layer.
+    depictionLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the depiction value chooser widget.
+    planeNormalButtons : PlaneNormalButtons
+        Buttons controlling plane normal orientation when the `plane` depiction value is choosed.
+    planeNormalLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the plane normal value chooser widget.
+    planeThicknessSlider : superqt.QLabeledDoubleSlider
+        Slider controlling plane normal thickness when the `plane` depiction value is choosed.
+    planeThicknessLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the plane normal thickness value chooser widget.
     renderComboBox : qtpy.QtWidgets.QComboBox
         Dropdown menu to select the rendering mode for image display.
     renderLabel : qtpy.QtWidgets.QLabel
         Label for the rendering mode dropdown menu.
+    attenuationSlider : qtpy.QtWidgets.QSlider
+        Slider controlling attenuation rate for `attenuated_mip` mode.
+    attenuationLabel : qtpy.QtWidgets.QLabel
+        Label for the attenuation slider widget.
+    isoThresholdSlider : qtpy.QtWidgets.QSlider
+        Slider controlling the isosurface threshold value for rendering.
+    isoThresholdLabel : qtpy.QtWidgets.QLabel
+        Label for the isosurface threshold slider widget.
     """
```

Comment:
What is the order of this section? Maybe it will be better to sort it alphabetically? 
It should be easier to search in it.

Reply:
I think the only order is that inherited attributes come first but sorting alphabetically makes sense, will do :+1: